doc/en/example/pythoncollection.rst

```diff
@@ -143,11 +143,9 @@ interpreters and will leave out the setup.py file::
 
     $ py.test --collect-only
```

Comment:
Is there a way to force this to be run on python 2?

The text above says:

> then a pytest run on python2 will find the one test when run with a python2 interpreters and will leave out the setup.py file::

and this example fails to demonstrate that now :wink: